src/pages/home/report/ReportActionItem.tsx

```diff
@@ -180,7 +193,7 @@ function ReportActionItem({
     const originalReportID = ReportUtils.getOriginalReportID(report.reportID, action);
     const originalReport = report.reportID === originalReportID ? report : ReportUtils.getReport(originalReportID);
     const isReportActionLinked = linkedReportActionID && action.reportActionID && linkedReportActionID === action.reportActionID;
-
+    const transactionCurrency = !lodashIsEmpty(transaction?.modifiedCurrency) ? transaction?.modifiedCurrency : transaction?.currency;
```

Comment:
Should we use [transactionUtils.getCurrency](https://github.com/Expensify/App/blob/cd18bef15f13683a6eaab33012770b3800027490/src/libs/TransactionUtils.ts#L286-L292) here?